modules/performing-an-image-signature-verification.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+//
+// * operating/verify-image-signatures.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="performing-an-image-signature-verification_{context}"]
+= Performing an image signature verification
+
+Before you perform an image signature verification, you must first create a signature integration in {rh-rhacs-first}.
+
+When you create a signature integration, you can use the following verification methods:
+
+* Cosign public keys
+* Cosign certificates
+* Transparency log
+
+.Prerequisites
+
+. You have a Cosign public key which is encoded in Privacy Enhanced Mail (PEM) format.
++ 
+For more information about Cosign public keys, see link:https://docs.sigstore.dev/cosign/signing/overview/[Overview] (Sigstore documentation).
+
+. You have the certificate identity and issuer.
+. Optional: You have a certificate and chain which is encoded in PEM format. 
++
+For more information about Cosign certificates, see link:https://docs.sigstore.dev/cosign/verifying/verify/[Verifying Signatures] (Sigstore documentation).
+
+.Procedure
+
+. In the {product-title-short} portal, click *Platform Configuration* -> *Integrations*.
+. Scroll down to the *Signature Integrations* section, and click *Signature*.
+. To create a new signature integration, click *New integration*.
+. Enter a name for the integration. You can save the integration only after you specify a name for the integration.
+. Optional: To add a new public key, complete the following steps:
```

Comment:
Adding a new public key should be a separate procedure module and all of this removed from the current module, especially since it is not required for image signature verification. We need to stick to the current procedure only here.